src/chrome/content/rules/Technical.ly.xml

```diff
@@ -1,25 +1,47 @@
 <ruleset name="Technical.ly (partial)">
-
 	<target host="technical.ly" />
 	<target host="www.technical.ly" />
-		<!--
-			These paths redirect to http:
-
-				- $
-				- 2013/03/12/founder-institute-boasts-helping-to-launch-750-startups-in-40-cities-infographic/$
-				- about/$
-				- advertising/$
-				- category/business/$
-				- contact-us/$
-				- directory/$
-				- ethics/$
-				- jobs/$
-				- organization/accessmd-com/$
-				- post-a-job/$
-							-->
-		<exclusion pattern="^http://(?:www\.)?technical\.ly/(?:$|\?|\d{4}/\d\d/\d\d/|about|advertising|category|contact-us|directory|ethics|jobs|organization|post-a-job)" />
 
+	<!-- 
+	These paths redirect to http:
+		- $
+		- 2013/03/12/founder-institute-boasts-helping-to-launch-750-startups-in-40-cities-infographic/$
+		- about/$
+		- advertising/$
+		- category/business/$
+		- contact-us/$
+		- directory/$
+		- ethics/$
+		- jobs/$
+		- organization/accessmd-com/$
+		- post-a-job/$
+	-->
+	<exclusion pattern="^http://(?:www\.)?technical\.ly/(?:$|\?|\d{4}/\d\d/\d\d/|about|advertising|category|contact-us|directory|ethics|jobs|organization|post-a-job)" />
```

Comment:
It's in our code guidelines:
`Prefer capturing groups (www\.)? over non-capturing (?:www\.)?. `

The `$` on the end is required. Otherwise it would also match `http://technical.ly/about/foo`